lib/app/modules/quality_reporting/measure_availability_sequence.rb

```diff
@@ -21,9 +21,21 @@ class MeasureAvailability < SequenceBase
           link 'https://www.hl7.org/fhir/measure.html'
           desc 'Check to make sure specified measure is available on test server'
         end
+
+        # Look for matching measure from cqf-ruler datastore by resource id
         measure_id = @instance.measure_to_test
-        query_response = @client.search(FHIR::Measure, search: { parameters: { _id: measure_id } })
+        measure_resource = @instance.module.measures.find { |m| m.resource.id == measure_id }
+
+        @client.additional_headers = { 'x-api-key': @instance.api_key, 'Authorization': @instance.auth_header }
```

Comment:
If `api_key` and `auth_header` aren't needed perhaps skip this.

Reply:
added a condition

Reply:
It doesn't look like you added a condition.

Reply:
oh whoops I only added it in the submit-data sequence. will add to the measure availability sequence too